### PR TITLE
Flakey tests in task pager

### PR DESCRIPTION
**Pull request by hschallhorn**
Fixes for [these](https://github.com/department-of-veterans-affairs/caseflow/issues/10516#issuecomment-523050358) flakes

Believe this is caused by duplicate values. Have been unable to reproduce locally.

### CODE REVIEW

**Comment by hschallhorn** [comment]
FINALLY
```
       expected: ["ARC", "Albuquerque", "Atlanta Education Center", "Columbia", "Detroit", "Digital Service HQ", "Hono...ster", "Manila", "Newark", "Philadelphia Insurance Center", "Philadelphia Pension Center", "VHA CO"]
            got: ["Albuquerque", "ARC", "Atlanta Education Center", "Columbia", "Detroit", "Digital Service HQ", "Hono...ster", "Manila", "Newark", "Philadelphia Insurance Center", "Philadelphia Pension Center", "VHA CO"]
```

File: spec/models/task_pager_spec.rb [hunk 1]
```diff
@@ -243,7 +243,7 @@
 
       it "sorts by regional office city" do
         expected_order = created_tasks.sort_by do |task|
-          RegionalOffice::CITIES[task.appeal.closest_regional_office][:city]
+          RegionalOffice::CITIES[task.appeal.closest_regional_office][:city].downcase
```
Comment (hschallhorn) [inline spec/models/task_pager_spec.rb]: Fixes [this difference](https://github.com/department-of-veterans-affairs/caseflow/pull/11842#issuecomment-523105545) in sorting

File: spec/models/task_pager_spec.rb [hunk 2]
```diff
@@ -272,8 +272,8 @@
 
       before do
         created_tasks.each do |task|
-          first_name = Faker::Name.first_name
-          last_name = "#{Faker::Name.middle_name} #{Faker::Name.last_name}"
+          first_name = Faker::Name.unique.first_name
+          last_name = "#{Faker::Name.unique.middle_name} #{Faker::Name.unique.last_name}"
```
Comment (hschallhorn) [inline spec/models/task_pager_spec.rb]: Ensures unique names so we don't get repeat values and sorting errors

File: spec/models/task_pager_spec.rb (hunk 1 above)
Comment (lomky) [reply spec/models/task_pager_spec.rb]: aaaah, ASCIIbetical vs alphabetical!

**Pull request merged**